paddlenlp/datasets/bq_corpus.py

```diff
@@ -0,0 +1,67 @@
+import collections
+import json
+import os
+
+from paddle.dataset.common import md5file
+from paddle.utils.download import get_path_from_url
+from paddlenlp.utils.env import DATA_HOME
+from . import DatasetBuilder
+
+__all__ = ['BQCorpus']
+
+
+class BQCorpus(DatasetBuilder):
+    """
+    BQCorpus: the largest dataset available for for the banking and finance sector
+
+    by frozenfish123@Wuhan University
```

Comment:
Please also give origanal author information.

Reply:
```
"""
    BQ Corpus (Bank Question Corpus), a large-scale domain-specific Chinese corpus for sentence semantic
    equivalence identification is constructed from online Webank custom service logs.

    The BQ corpus contains 120,000 question pairs with manual annotation.

    The following two lines are examples extracted from training data:

        '''
        "微粒贷开通"	你好，我的微粒贷怎么没有开通呢	0
        为什么借款后一直没有给我回拨电话    怎么申请借款后没有打电话过来呢！    1
        '''

    Provider: Intelligent Computing Research Center, Harbin Institute of Technology(Shenzhen)
    Contacts: Qingcai Chen (email: qingcai.chen@hit.edu.cn; Fax: +86-755-26033182)
    More Info: https://www.luge.ai/

     """
```

Sorry I have been preparing for my final exam these days and I didn't reply timely. Here are my chages on this opinion.

Reply:
Great description! Good luck in the final.